### ISSUE TITLE
Sonderzeichen für Typbezeichnungen festlegen

### ISSUE THREAD

**Issue by baeuchle** [baeuchle/ds100bot#42]
Neben
# für Orte
$ für Strecken
% für Signale
& für allgemeine Abkürzungen
/ für Linien
soll es noch ein anderes Sonderzeichen geben, das Typen vorangestellt werden kann, z.B. ^FFM:U2 für den U2-Wagen aus Frankfurt, oder ~BR120 für die erste Drehstromserienlok der Welt, oder !Bvmz oder so.

Diskussion zu einem passenden Zeichen erwünscht!

**Comment by NiklasDah** [comment]
Wie wäre ein „:“? Das wäre wohl auf allen Geräten leicht zugänglich. Sonst würde ich als iOS-User „!“ oder „~“ vorschlagen, die sind aber wohl auf anderen Tastaturen ziemlich versteckt. „^“ wäre für PC-User nervig.

**Comment by baeuchle** [comment]
: ist Teil der anderen Suchbegriffe, als #FFM:HB. Daher eher nicht.

**Comment by baeuchle** [comment]
Meine Gedanken, unsortiert:
°Bvmz doof wegen °C
^111 doof wegen Potenz
=U2 doof wegen Gleichheit
~M doof wegen ungefähr
\U2 ...? Vielleicht ok.
*U2 doof wg *grins* oder *korrektur

**Comment by NiklasDah** [comment]
> : ist Teil der anderen Suchbegriffe, als #FFM:HB. Daher eher nicht.

Naja, wenn man ein vorhergehendes Leerzeichen voraussetzt, müsste es ja funktionieren. Sonst ein „;“?

**Comment by Jonaes** [comment]
Via Twitter @tk_wro https://twitter.com/TK_Wro/status/1287352992342200320?s=19

"~ für ~BR ist bei GBoard ziemlich versteckt. Ein *BR oder -BR wäre zumindest für Android angenehmer 😉"

**Comment by baeuchle** [comment]
> > : ist Teil der anderen Suchbegriffe, als #FFM:HB. Daher eher nicht.
> 
> Naja, wenn man ein vorhergehendes Leerzeichen voraussetzt, müsste es ja funktionieren. Sonst ein „;“?

Stimmt, ich setze definitiv schon eine Wortgrenze voraus. Dann bleibt noch ein Problem mit Links, das hatte ich auch anfangs bei "/" für Linien (und dort, immerhin, gelöst).

**Comment by baeuchle** [comment]
Danke schonmal für die Kommentare. Ich mache mir weiter Gedanken. Währenddessen arbeite ich mal die ganzen tollen Listen ab, die sich hier angesammelt haben.

**Comment by baeuchle** [comment]
> Stimmt, ich setze definitiv schon eine Wortgrenze voraus. Dann bleibt noch ein Problem mit Links, das hatte ich auch anfangs bei "/" für Linien (und dort, immerhin, gelöst).

Mir fällt auf: der fix für / in links würde das hier auch schon fixen. Der Kommentar war dumm.

**Comment by baeuchle** [comment]
Beim drübernachdenken nach einem halben Jahr finde ich ";" ganz gut.
